templates/downloads/os_list.html

```diff
@@ -35,6 +35,11 @@ <h1 class="page-title">Python Releases for {{ os.name }}</h1>
 
             <li><a href="{{ latest_python3.get_absolute_url }}">Latest Python 3 Release - {{ latest_python3.name }}</a></li>
         </ul>
+
+        {% if os.slug == 'android' %}
+        <p>Rather than using these packages directly, in most cases you should use one of the tools recommended in the <a href="https://docs.python.org/3/using/android.html">Python documentation</a>.</p>
```

Comment:
```suggestion
        <p>Rather than using these packages directly, in most cases you should use one of the tools recommended in the <a href="https://docs.python.org/3/using/android.html">Python documentation</a>.</p> 
```

Reply:
just needing to kick off ci to see if terraform ci will be happy with changes from prefix to path mode